south-african-journal-of-animal-science.csl

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
   <info>
-    <title>South African Journal of Animal Science</title>
+    <title>south-african-journal-of-animal-science.csl</title>
```

Comment:
The title should have remained `<title>South African Journal of Animal Science</title>`.